frontend/src/metabase/embedding/embedding-iframe-sdk-setup/components/SelectEmbedOptionsStep.tsx

```diff
@@ -262,6 +263,41 @@ const BehaviorSection = () => {
                 />
               )}
             </WithNotAvailableWithoutSimpleEmbeddingFeatureWarning>
+
+            <WithNotAvailableForGuestEmbedsWarning
+              campaign={UPSELL_CAMPAIGN_BEHAVIOR}
+            >
+              {({ disabled: disabledInGuestEmbedding }) => {
+                return (
+                  <Flex align="center" gap="xs">
+                    <Checkbox
+                      disabled={!hasEmailSetup || disabledInGuestEmbedding}
+                      label={t`Allow subscriptions`}
+                      checked={settings.withSubscriptions}
+                      onChange={(e) =>
+                        updateSettings({ withSubscriptions: e.target.checked })
+                      }
+                    />
+                    {!hasEmailSetup && !disabledInGuestEmbedding && (
```

Comment:
The other hover card is controlled within `WithNotAvailableForGuestEmbedsWarning`, so I have another hover card here that only concerns email not set up.